src/layouts/Resources.jsx

```diff
@@ -61,79 +112,108 @@ export default class Resources extends Component {
     }
```

Comment:
I didn't catch this in the previous PR but since we are using `bluebird`'s `Promise.map` function, wouldn't the returned resources be sorted by cateory?